students/11th/hyojeung/style/login.css

```
@@ -0,0 +1,73 @@
+/* mainBox */
+.warp{
+  width: 1200px;
+  height: 100%;
+  margin: 0 auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;}
+
+.mainBox{
+  width: 350px;
+  height: 379px;
+  border: 1px solid #e6e6e6;
+  border-radius: 1px;
+  background: #fff;
+  margin: 0 auto;}
+
+.logo{
+  text-align: center;
+  margin: 35px auto 25px;}
+
+.loginBox{
+  width: 350px;
+  height: 170px;
+  display: flex;
+  text-align: center;
+}
+.loginFom{
+  justify-content: center;
+  flex-direction: column;}
+
+.loginFom input{
+    width: 260px;
+    height: 36px;
+    font-size: 12px;
+    letter-spacing: -1px;
+    color: #8e8e8e;
+    background-color: #fafafa;
+    border: 1px solid #dddddd;
+    padding: 3px 0 0 10px;
+    border-radius: 5px;
+    line-height: 36px;
+    margin-bottom: 5px;}
```

```suggestion
.loginForm input{
    width: 260px;
    height: 36px;
    font-size: 12px;
    letter-spacing: -1px;
    line-height: 36px;
    color: #8e8e8e;
    background-color: #fafafa;
    border: 1px solid #dddddd;
    border-radius: 5px;
    padding: 3px 0 0 10px;
    margin-bottom: 5px;
}
```
1. 중요한 속성을 위로, 유사한 속성끼리 붙여서 사용해주세요
2. 오타 수정해주세요

---

1. border-radius와 line-height 위치변경하여 유사속성끼리 수정하였습니다.
2. loginFom 오타 loginForm으로 수정완료하였습니다. 

아래부터 수정해나가고 있습니다! 댓글을 달면서 수정하고 완료후 한번에 add하도록하겠습니다!